editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -637,7 +637,7 @@ void CanvasItemEditor::_find_canvas_items_at_pos(const Point2 &p_pos, Node *p_no
 
 			_SelectResult res;
 			res.item = c;
-			res.z = node ? node->get_z() : 0;
+			res.z = node ? node->get_z_index() : 0;
```

Comment:
Maybe `z` should also be renamed to `z_index` in `_SelectResult` for consistency?

Reply:
Right, it's modified !